scripts/variantstore/wdl/GvsCreateVAT.wdl

```diff
@@ -604,3 +619,72 @@ task BigQueryExportVat {
     }
 }
 
+task MergeVatTSVs {
+    input {
+        Array[Boolean] export_done
+        Array[String] contig_array
+        String project_id
+        String output_path
+
+        Int? merge_vcfs_disk_size_override
+        String? service_account_json_path
+    }
+
+    Int disk_size = if (defined(merge_vcfs_disk_size_override)) then merge_vcfs_disk_size_override else 50
+    String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+
+    command <<<
+        apt-get update
```

Comment:
I think the latest gatk docker (us.gcr.io/broad-gatk/gatk:4.2.6.1) has gsutil, tabix, etc in it.